app/controllers/item.js

```diff
@@ -72,6 +73,14 @@ export default Ember.Controller.extend({
     return this.store.peekRecord("package_category", this.get("categoryId"));
   }),
 
+  linkDisplayName: Ember.computed("prevPath", function() {
```

Comment:
could we move that to the template file ?
That way we keep data and rendering separate :)

```
{{#if (is-equal prevPath "search_goods")}}
     {{t 'search_goods.back'}}
{{else}}
    {{ categoryObj.name }}
{{/if}
```

Reply:
Yeah, it can be done, but i have purposely done that to separate out logic from the view files to the controller.
Let me know, if i should change it !